src/lib/y2remote/modes/manager.rb

```diff
@@ -1,25 +1,65 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2017 SUSE LLC
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE.
+#
+# To contact SUSE about this file by physical or electronic mail, you may find
+# current contact information at www.suse.com.
+# ------------------------------------------------------------------------------
+
 require "y2remote/modes/base"
 
 module Y2Remote
   module Modes
+    # This class is reponsible of vncmanager service management
     class Manager < Base
+      # <String> Service name
```

Comment:
I wolud use `[String]` (just for consistence). Even `@return [String]` if you want.